tests/attacks/inference/attribute_inference/test_true_label_baseline.py

```diff
@@ -308,8 +314,8 @@ def test_true_label_baseline_regression(art_warning, get_diabetes_dataset, model
             baseline_inferred_test
         )
 
-        assert 0.6 <= baseline_train_acc
-        assert 0.6 <= baseline_test_acc
+        assert 0.45 <= baseline_train_acc
```

Comment:
Yes, I think this was the reason for reducing the value. How do you suggest to deal with it?

Reply:
I was thinking of it could be worth having a dictionary of expectation value like
```python
expected_accuracy = {"nn": 0.65, "rf": 0.45, ...}
```
and use it like
```python
assert expected_accuracy[model_type] == baseline_train_acc 
```